Lombiq.Hosting.Tenants.IdleTenantManagement.Tests.UI/Extensions/IdleTenantManagementExtensions.cs

```diff
@@ -26,5 +27,5 @@ public static class IdleTenantManagementExtensions
     public static readonly Func<IWebApplicationInstance, Task> AssertAppLogsWithIdleCheckAsync =
         async webApplicationInstance =>
             (await webApplicationInstance.GetLogOutputAsync())
-            .ShouldContain("Shutting down tenant \"Default\" because of idle timeout.");
+            .ShouldContain($"Shutting down tenant \"{IdleTenantShutdown}\" because of idle timeout.");
```

Comment:
What I've meant here is that the function should have a parameter that is used for the tenant name, just like in TestIdleTenantManagerBehaviorAsync. Because in TestIdleTenantManagerBehaviorAsync it's dynamic and here it is constant.